From 948a660e3e80d34e8509f5e3e0dc8701b4491763 Mon Sep 17 00:00:00 2001
From: Cesare Mercurio <cesare.mercurio@gmail.com>
Date: Sun, 17 Dec 2023 07:57:48 -0800
Subject: [PATCH 1/2] VVI-node-api-headers : patch for node-api-headers module
 bzlmod

Summary:
Add module.bazel file for bzlmod registry

Test Plan:
integration with vvplatform
---
 MODULE.bazel | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 MODULE.bazel

diff --git a/MODULE.bazel b/MODULE.bazel
new file mode 100644
index 0000000..5c10559
--- /dev/null
+++ b/MODULE.bazel
@@ -0,0 +1,6 @@
+module(
+    name = "node-api-headers",
+    version = "1.1.0",
+    compatibility_level = 1,
+)
+bazel_dep(name = "rules_cc", version = "0.0.5")
\ No newline at end of file
-- 
2.34.1

